doc/website/getting-started/installation.md

```diff
@@ -4,6 +4,26 @@ iceoryx_utils and iceoryx_posh are deployed as independent cmake packages. Posh
 
 ## Prerequisites
 
+### General
```

Comment:
What do you think about naming this section "Dependencies" or "General dependencies"?

Reply:
Named it Dependencies